ca/certificates.go

```diff
@@ -119,6 +119,13 @@ type RootCA struct {
 	// must correspond to the key in the local signer, if provided
 	Cert []byte
 
+	// Intermediates contains a bundle of PEM encoded certificates to append to any issued TLS certificates.
+	// If more than one are provided, the first one must have the same public key and subject as the signing
+	// root certificate (or, if no key is provided, at least one of the root certificates), and the rest
```

Comment:
Maybe we add little ASCII Diagrams? We will sendup with a certificate that contains: [Leaf] | [Intermediate] and a root that contains [Root]. We guarantee that there is always a valid chain [Leaf]  to [Root] using [Intermediate].

😄